src/scenes/CommunityPage/CommunityPageView.tsx

```diff
@@ -147,19 +147,12 @@ const StyledCommunityPageContainer = styled.div`
 `;
 
 const StyledBaseM = styled(BaseM)<{ showExpandedDescription: boolean }>`
-  -webkit-line-clamp: ${({ showExpandedDescription }) => (showExpandedDescription ? 'initial' : 4)};
+  -webkit-line-clamp: ${({ showExpandedDescription }) => (showExpandedDescription ? 'initial' : 5)};
   display: -webkit-box;
   -webkit-box-orient: vertical;
   overflow: hidden;
-  line-clamp: 4;
+  line-clamp: 5;
```

Comment:
would look like this otherwise

<img width="613" alt="image" src="https://user-images.githubusercontent.com/12162433/208751469-1bf6f2a1-80ba-4cf0-9fe5-dbf9e5cdee11.png">